Multiplatform/Collections/Artists/ArtistListRow.swift

```diff
@@ -21,6 +24,21 @@ struct ArtistListRow: View {
                 Text(artist.name)
                     .padding(.horizontal, 5)
             }
+            .onAppear { expand?() }
```

Comment:
Why make this part of the artist row and not just attach the `onAppear` modifier in `ArtistList.swift`?

Reply:
Because you told me to do it like this?

> This modifier can just be added to the normal artist row

Reply:
I meant here in the `ArtistsList.swift` https://github.com/rasmuslos/AmpFin/blob/d906768b47aa9c58a2cf9e15ea74f556840ac561/Multiplatform/Collections/Artists/ArtistList.swift#L20 Sorry if I have not expressed that clearly. Adding an additional parameter seems kind of unnecessary.